config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -43,6 +43,8 @@ presubmits:
         - --cluster=
         - --env=HEAPSTER_MACHINE_TYPE=e2-standard-8
         - --env=KUBEMARK_APISERVER_TEST_ARGS=--max-requests-inflight=80 --max-mutating-requests-inflight=0 --profiling --contention-profiling
+        - --env=KUBE_COS_INSTALL_CONTAINERD_VERSION=v2.0.5
+        - --env=KUBE_COS_INSTALL_RUNC_VERSION=v1.2.1
```

Comment:
I think this is OK since this job is not blocking anyhow, but we should be prepared to rollback.